DOCUMENTATION.md

```diff
@@ -1,810 +1,780 @@
+  name: "John Smith",
+  year: "2016"                      
+}
+```
 
+You can leave out this argument if a template does not have any placeholder fields for personalised information.
 
-#### Arguments
+#### reference (optional)
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+A unique identifier. This reference identifies a single unique notification or a batch of notifications.
 
-##### `email_address`
-The email address of the recipient, only required for email notifications.
+```ruby
+reference: "your_reference_string"
+```
 
-##### `template_id`
+You can leave out this argument if you do not have a `reference`.
 
-Find by clicking **API info** for the template you want to send.
+#### email_reply_to_id (optional)
 
-##### `reference`
+This is an email reply-to address specified by you to receive replies from your users. Your service cannot go live until you set up at least one of these email addresses. To set up:
 
-An optional identifier you generate. The `reference` can be used as a unique reference for the notification. Because Notify does not require this reference to be unique you could also use this reference to identify a batch or group of notifications.
+1. Sign into your GOV.UK Notify account.
+1. Go to __Settings__.
+1. If you need to change to another service, select __Switch service__ in the top right corner of the screen and select the correct one.
+1. Go to the Email section and select __Manage__ on the __Email reply to addresses__ row.
+1. Select __Change__ to specify the email address to receive replies, and select __Save__.
 
-You can omit this argument if you do not require a reference for the notification.
+```ruby
+email_reply_to_id: '8e222534-7f05-4972-86e3-17c5d9f894e2'
+```
 
-##### `email_reply_to_id`
+You can leave out this argument if your service only has one email reply to address, or you want to use the default email address.
 
-Optional. Specifies the identifier of the email reply-to address to set for the notification. The identifiers are found in your service Settings, when you 'Manage' your 'Email reply to addresses'.
+### Response
 
-If you omit this argument your default email reply-to address will be set for the notification.
+If the request to the client is successful, the client will return a `Notifications::Client:ResponseNotification` object. This object is named `emailresponse` in the method example.
 
-##### `personalisation`
+You can then call different methods on this object to return the requested information:
 
-If a template has placeholders, you need to provide their values, for example:
+|Method|Information|Type|
+|:---|:---|:---|
+|`emailresponse.id`|Notification UUID|String|
+|`emailresponse.reference`|`reference` argument|String|
+|`emailresponse.content`|- Message body<br>- Message subject<br>- From email address of your service found on the **Settings** page|Hash|
+|`emailresponse.template`|Contains the `id`, `version` and `URL` of the template|Hash|
```

Comment:
The URL in the hash is labelled as `uri` so the code snippet here needs to read `uri` (has to be lower case too)